data-prepper-plugins/aws-lambda/README.md

```diff
@@ -0,0 +1,72 @@
+
+# Lambda Processor
```

Comment:
remove processor doc?

Reply:
Let's keep the developer guide for running integ tests here. We can remove the usage sections after they are added to Opensearch documentation website.

Reply:
do we want to have 2 different readme for lambda processor and sink?